packages/playground/windows/playground/App.cpp

```diff
@@ -0,0 +1,275 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.#include "pch.h"
+
+#include "ReactPackageProvider.h"
+
+using namespace winrt::playground;
+using namespace winrt::playground::implementation;
+
+using namespace winrt;
+using namespace Windows::ApplicationModel::Activation;
+using namespace Windows::Foundation;
+using namespace Windows::UI;
+using namespace Windows::UI::Xaml;
+using namespace Windows::UI::Xaml::Controls;
+using namespace Windows::UI::Xaml::Media;
+using namespace Windows::Storage;
+using namespace Windows::Storage::Streams;
+using namespace Windows::Graphics::Imaging;
+using namespace Windows::Storage::Pickers;
+
+struct App : ApplicationT<App> {
+  void OnLaunched(LaunchActivatedEventArgs const &) {
+    const hstring xamlString =
+        L" <Grid"
+        L"  xmlns='http://schemas.microsoft.com/winfx/2006/xaml/presentation'"
+        L"  xmlns:x='http://schemas.microsoft.com/winfx/2006/xaml'>"
+        L"   <Grid.RowDefinitions>"
+        L"     <RowDefinition Height='auto'/>"
+        L"     <RowDefinition Height='auto'/>"
+        L"     <RowDefinition Height='auto'/>"
+        L"     <RowDefinition Height='*'/>"
+        L"   </Grid.RowDefinitions>"
+        L"   <Grid.ColumnDefinitions>"
+        L"     <ColumnDefinition Width='auto'/>"
+        L"     <ColumnDefinition Width='*'/>"
+        L"   </Grid.ColumnDefinitions>"
+        L""
+        L"   <Button x:Name='x_LoadButton'"
+        L"     Grid.Column='0'"
+        L"     Grid.RowSpan='2'"
+        L"     Margin='2,2,2,2'"
+        L"     VerticalAlignment='Stretch'"
+        L"     Content='Load' />"
+        L""
+        L"   <Grid Grid.Row='0' Grid.Column='1'>"
+        L"     <Grid.ColumnDefinitions>"
+        L"       <ColumnDefinition Width='auto'/>"
+        L"       <ColumnDefinition Width='*'/>"
+        L"       <ColumnDefinition Width='auto'/>"
+        L"       <ColumnDefinition Width='auto'/>"
+        L"     </Grid.ColumnDefinitions>"
+        L""
+        L"     <TextBlock"
+        L"       Grid.Column='0'"
+        L"       Margin='2,2,2,2'"
+        L"       VerticalAlignment='Center'"
+        L"       Text='JavaScript File (no extension):'/>"
+        L"     <ComboBox x:Name='x_JavaScriptFilename'"
+        L"       Grid.Column='1'"
+        L"       Margin='2,2,2,2'"
+        L"       MinWidth='250'"
+        L"       SelectedIndex='0'"
+        L"       VerticalAlignment='Center'"
+        L"       HorizontalAlignment='Stretch'>"
+        L"       <ComboBox.Items>"
+        L"         <ComboBoxItem Content='Samples\\rntester' />"
```

Comment:
Ideally this would be databound instead of hardcoding a set of known items

Reply:
I'll leave this as a future improvement for now.  The current playground doesn't do this.